### PR TITLE
docs: update to link emoji in README.md

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

Change Firebase logo to link emoji.

## Related Issues

none.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [X] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,39 +44,39 @@ and open source.
 
 | Name                   | pub.dev                                                                                                                                             | Firebase Product                                                                                                                                                             | Documentation                                                     | View Source                                                                                                                     | Android | iOS | Web | MacOS |
 |------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|---------|-----|-----|-------|
```
Comment (Salakar) [inline README.md:46]:
```suggestion
|------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------:|:---------:|:-----:|:-----:|-------:|
```

Centering the columns

**Review by Salakar — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -43,40 +43,40 @@ and open source.
...
 
 ## Beta Plugins
 
 | Name      | pub.dev                                                                                                                        | Firebase Product                                                                                                                                         | Documentation                                              | View Source                                                                                                     | Android | iOS | Web | MacOS |
 |-----------|--------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|---------|-----|-----|-------|
```
Comment (Salakar) [inline README.md:65]:
```suggestion
|-----------|:--------------------------------------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------------------------:|:---------:|:-----:|:-----:|:-------:|
```

File: README.md [hunk 3]
```diff
@@ -43,40 +43,40 @@ and open source.
...
 
 
 ## Preview Plugins
 
 | Name                | pub.dev                                                                                                                                                      | Firebase Product                                                                                                                                      | Documentation                                                        | View Source                                                                                                                         | Android | iOS | Web | MacOS |
 |---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|---------|-----|-----|-------|
```
Comment (Salakar) [inline README.md:72]:
```suggestion
|---------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------:|:---------:|:-----:|:-----:|:-------:|
```

File: README.md [hunk 4]
```diff
@@ -43,40 +43,40 @@ and open source.
...
 
 ## Alpha Plugins
 
 | Name                | pub.dev                                                                                                                                    | Firebase Product                                                                                                                          | Documentation                                                  | View Source                                                                                                       | Android | iOS | Web | MacOS |
 |---------------------|--------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|---------|-----|-----|-------|
```
Comment (Salakar) [inline README.md:78]:
```suggestion
|---------------------|:--------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------:|:---------:|:-----:|:-----:|:-------:|
```

**Review by Lyokone — APPROVED** [review]

**Review by pr-Mais — APPROVED** [review]

**Pull request merged**